examples/02_first_level_models/plot_adhd_dmn.py

```diff
@@ -0,0 +1,76 @@
+"""
+Default Mode Network extraction of AHDH dataset
+===============================================
+
+Full step-by-step example of fitting a GLM to a seed extract on the
+Posterior Cingulate Cortex data and saving the results.
+
+More specifically:
+
+1. A sequence of fMRI volumes are loaded
+2. A design matrix with the Posterior Cingulate Cortex seed is defined
+4. A GLM is applied to the dataset (effect/covariance,
+   then contrast estimation)
+5. The Default Mode Network is display
+
+"""
+import numpy as np
+
+from nilearn import datasets, plotting
+from nilearn.input_data import NiftiSpheresMasker, NiftiMasker
+
+from nistats.first_level_model import FirstLevelModel
+from nistats.design_matrix import make_design_matrix
```

Comment:
In examples, we tend to put the imports only where they are needed. This is more natural for non-expert users.